pupil-spa/src/app/login/login.component.spec.ts

```diff
@@ -94,7 +94,7 @@ describe('LoginComponent', () => {
       promiseHelper.resolve({ success: 'login okay' });
     });
 
-    it('should initialise the QuestionService and WarmupQuestionService on login', async () => {
+    xit('should initialise the QuestionService and WarmupQuestionService on login', async () => {
```

Comment:
The tests are ignored because there are no server calls now — there is no way to log in and take the test as normal; the normal flow is through the survey.

The pending submission screen is ‘replaced’ with `ict-survey/preview-completed`.